src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -0,0 +1,600 @@
+```bash
+mkdir terragrunt-config && cd terragrunt-config
+```
+In the new directory, create a `terragrunt.hcl` file, create a `./src` directory, and create a `./environments` directory:
+```bash
+touch terragrunt.hcl
+mkdir src
+mkdir environments
+```
+Inside of your `environments` folder, create a folder named `dev`.
+In the `./src` directory, create a `main.tf` and `provider.tf` files:
+```bash
+mkdir environments/dev
+touch src/main.tf
+touch src/provider.tf
+```
+
+In the __provider.tf__ file add the appropriate Terraform and New Relic provider configuration information:
+```hcl
+terraform {
+	required_version = "~> 0.14.3"
+	required_providers {
+		newrelic = {
+			source = "newrelic/newrelic"
+			version = "~> 2.14.0"
+		}
+	}
+}
+```
+
+In the __main.tf__ add a New Relic alert policy named DemoPolicy:
+```hcl
+resource "newrelic_alert_policy" "DemoPolicy" {
+	name = "My Demo Policy"
+}
+```
+
+In the `./environments/dev` directory create a new file named `terragrunt.hcl`
+```bash
+cd ./environments/dev/
+touch terragrunt.hcl
+```
+
+Within the dev folder update the `terragrunt.hcl` file by adding the following include statement. The include statement tells Terragrunt to use any `.hcl` confgiuration files it finds in parent folders.
+```hcl
+include {
+	path= find_in_parent_folders()
+}
+```
+
+Next, add a Terraform block to provide Terragrunt with a Terraform source reference.
+```hcl
+terraform {
+	source = "../../src"
+}
+```
+
+Now you need to configure the provider with your New Relic account details. You will need your [New Relic user API key](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher) and account ID.
+```hcl
+provider "newrelic" {
+	account_id = 12345 # Your New Relic account ID
+	api_key = "NRAK-***" # Your New Relic user key
+	region = "US" # US or EU (defaults to US)
+}
+```
+
+<Callout variant="tip">
+
+If you have a single account: The account ID is displayed on the API keys page.
+
+If your organization has multiple accounts: From [one.newrelic.com](https://one.newrelic.com/), select Entity explorer and use the account selector dropdown near the top of the page.
+
+</Callout>
+
+<Callout variant="tip">
+
+Documentation about the New Relic provider is provided on the [Terraform doc site.](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs)
+
+</Callout>
+
+Update the __provider.tf__ file to define the New Relic account variables that will help keep the Terraform configuration dynamic.
+```hcl
+variable "newrelic_personal_apikey" {}
+variable "newrelic_account_id" {}
+variable "newrelic_region" {}
+
+provider "newrelic" {
+	account_id = var.newrelic_account_id
+	api_key = var.newrelic_personal_apikey
+	region = var.newrelic_region
+}
+```
+
+In your terminal, change directories to the environments/dev folder and run the command `terragrunt init`:
+```bash
+terragrunt init
+```
+
+Terraform init sets up a bit of context, sets up environment variables, and then runs `terraform init`.
```

Comment:
Would it make sense to say something like, "You will see the following output:" after this sentence?